csharp/sample/Microsoft.ML.OnnxRuntime.InferenceSample/Program.cs

```diff
@@ -39,15 +39,15 @@ static void UseApi()
                 }
 
                 // Run the inference
-                var results = session.Run(container);  // results is an IReadOnlyList<NamedOnnxValue> container
-
-                // dump the results
-                foreach (var r in results)
+                using (var results = session.Run(container))  // results is an IReadOnlyList<NamedOnnxValue> container
```

Comment:
(nit) the comment is now incorrect.  `results` is `IDisposableReadOnlyCollection<DisposableNamedOnnxValue>`

Reply:
An easy "fix" for this is to delete the comment and not use `var`. Instead use the actual type when declaring the `results` variable.

---
In reply to: [251587744](https://github.com/Microsoft/onnxruntime/pull/392#discussion_r251587744) [](ancestors = 251587744)